minecraft_servers/awantalife/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "server_name": "awantalife",
+    "nice_name": "AwantaLife.de Citybuild Netzwerk",
+    "direct_ip": "awantalife.de",
+    "server_wildcards": [
+      "%.awantalife.de"

```

Comment:
Not a real wildcard
https://i.imgur.com/bDHUkLW.png

Reply:
Wildcard Fixed